packages/contracts/test/core/permission/permission-manager.ts

```diff
@@ -642,62 +642,62 @@ describe('Core: PermissionManager', function () {
     });
   });
 
-  describe('hasPermissions', () => {
+  describe('isGranted', () => {
     it('should return true fcr granted user', async () => {
```

Comment:
```suggestion
    it('should return true if the permission is granted to the user', async () => {
```
Maybe this is an opportunity to improve the other test descriptions that are poorly written.